src/components/apiDelegation/given/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -6,73 +6,155 @@ import {
   ButtonVariant,
   ButtonColor,
   ButtonSize,
+  SearchField,
+  Panel,
+  PanelVariant,
 } from '@altinn/altinn-design-system';
 import type { Key } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 import * as React from 'react';
+import { useState, useEffect } from 'react';
 
-import { softAddOrg, softRemoveOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { ActionBar, ActionIconVariant } from '@/components/reusables';
+import {
+  softAddOrg,
+  softRemoveOrg,
+  searchInCurrentOrgs,
+  lookupOrg,
+} from '@/rtk/features/delegableOrg/delegableOrgSlice';
 import { useAppDispatch, useAppSelector } from '@/rtk/app/hooks';
 import type { DelegableOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
 import { ReactComponent as ApiIcon } from '@/assets/ShakeHands.svg';
-import {
-  NewDelegationAccordion,
-  NewDelegationAccordionButtonType,
-} from '@/components/reusables/NewDelegationAccordion';
 import { RouterPath } from '@/routes/Router';
 import { PageContainer } from '@/components/reusables/PageContainer';
 import main from '@/main.module.css';
 
 import classes from './ChooseOrgPage.module.css';
 
 export const ChooseOrgPage = () => {
-  const delegableOrgs = useAppSelector((state) => state.delegableOrg.delegableOrgList);
+  const delegableOrgs = useAppSelector((state) => state.delegableOrg.presentedOrgList);
   const chosenOrgs = useAppSelector((state) => state.delegableOrg.chosenDelegableOrgList);
+  const searchOrgNotExist = useAppSelector((state) => state.delegableOrg.searchOrgNonexistant);
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
+  const [searchString, setSearchString] = useState('');
+  const [promptOrgNumber, setPromptOrgNumber] = useState(false);
+  const fetchLookupOrg = async () => await dispatch(lookupOrg(searchString));
 
   const { t } = useTranslation('common');
 
-  const delegableApiAccordions = delegableOrgs.map((org: DelegableOrg, index: Key) => {
+  const IsOnlyNumbers = (str: string) => /^\d+$/.test(str);
+
+  const handleSoftRemove = (org: DelegableOrg) => {
+    dispatch(softRemoveOrg(org));
+    dispatch(searchInCurrentOrgs(searchString));
+  };
+
+  function handleSearch(searchText: string) {
+    if (IsOnlyNumbers(searchText) || searchText === '') {
```

Comment:
I get this when searching for digdir org nr: 
![image](https://user-images.githubusercontent.com/31471142/217816554-2a504c91-003c-4928-a81f-bb80dd127eb1.png)

It is probably because the response from backend is empty. Talk to backend about returning something else when no results. Also edit code to handle weird backend responses.

Reply:
Yeah, the lookup-api seems to have broken at some point over the last couple of weeks, cause it doesn't return anything anymore 😅
I've let back-end know there's a bug and they're on it 🙂